rules/xcodeproj.bzl

```diff
@@ -417,6 +417,9 @@ mkdir -p $BAZEL_DIAGNOSTICS_DIR
 export DATE_SUFFIX="$(date +%Y%m%d.%H%M%S%L)"
 export BAZEL_BUILD_EVENT_TEXT_FILENAME="$BAZEL_DIAGNOSTICS_DIR/build-event-$DATE_SUFFIX.txt"
 export BAZEL_BUILD_EXECUTION_LOG_FILENAME="$BAZEL_DIAGNOSTICS_DIR/build-execution-log-$DATE_SUFFIX.log"
+if [ "$PLATFORM_NAME" = "iphoneos" ]; then
```

Comment:
could this logic be moved entirely into the build wrapper?

Reply:
Yes. I put it here to make build-wrapper.sh agnostic to the xcode behavior, but probably it should not be in xcodeproj.bzl since it doesn't use any knowledge of xcodeproj.bzl.

Reply:
Done